test/test_datasets_video_utils.py

```diff
@@ -6,28 +6,27 @@
 from torchvision import io
 from torchvision.datasets.video_utils import VideoClips, unfold
 
-from common_utils import get_tmp_dir, assert_equal
+from common_utils import assert_equal
 
 
 @contextlib.contextmanager
-def get_list_of_videos(num_videos=5, sizes=None, fps=None):
-    with get_tmp_dir() as tmp_dir:
-        names = []
-        for i in range(num_videos):
-            if sizes is None:
-                size = 5 * (i + 1)
-            else:
-                size = sizes[i]
-            if fps is None:
-                f = 5
-            else:
-                f = fps[i]
-            data = torch.randint(0, 256, (size, 300, 400, 3), dtype=torch.uint8)
-            name = os.path.join(tmp_dir, "{}.mp4".format(i))
-            names.append(name)
-            io.write_video(name, data, fps=f)
-
-        yield names
+def get_list_of_videos(tmpdir, num_videos=5, sizes=None, fps=None):
```

Comment:
Oh, I'm discovering now that this function is duplicated. Let's get rid of both context manager (in this file and the other one above) and instead put the function version that I suggested in `common_utils.py`.

Reply:
Thanks, edited. :)